cudatoolkit-dev/cuda-install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash 
+
+TEMPDIR=/tmp/cuda 
+mkdir -p $TEMPDIR
+
+wget -c --directory-prefix=$TEMPDIR --quiet https://developer.nvidia.com/compute/cuda/9.2/Prod/local_installers/cuda_9.2.88_396.26_linux
+cuda_filepath=$TEMPDIR/"cuda_9.2.88_396.26_linux"
```

Comment:
Should be `="$TEMPDIR/cuda_9.2.88_396.26_linux`

Reply:
Also, this script should use the `$PKG_VERSION` environment variable [as set here](https://conda.io/docs/user-guide/tasks/build-packages/link-scripts.html) so that the version number isn't hard-coded here.